Documentation/configuration/metrics.rst

```diff
@@ -105,6 +105,12 @@ Controllers
 * ``controllers_runs_duration_seconds``: Duration in seconds of the controller
   process labeled by completion status
 
+Subroutines
+-----------
+
+* ``subroutine_start_total``: Number of times that a subroutine has started by
```

Comment:
Subroutine might give the impression that is a go routine, I would use "child processes" or "sub-processes"?